tests/Exception/ErrorException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace DrevOps\Robo\Tests\Exception;
+
+use PHPUnit\Framework\Exception;
+
+/**
+ * A new ErrorException class.
+ */
+class ErrorException extends Exception
```

Comment:
PHPunit 10 does not have PHPUnit\Framework\Error\Error.
I created this class to solve the missing class issue.